src/components/panel/examples/warn-branded/index.njk

```diff
@@ -0,0 +1,14 @@
+---
+"theme": census
```

Comment:
```suggestion
"theme": census
"fullWidth": true
```
This will make the example fill the width of the page which I think will make it look more how you want it to